storage/test/storage_suite.go

```diff
@@ -326,6 +329,16 @@ func (s *BaseStorageSuite) TestSetConfigInvalid(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (s *BaseStorageSuite) TestModule(c *C) {
+	storer, err := s.Storer.Module("foo")
+	c.Assert(err, IsNil)
+	c.Assert(storer, NotNil)
+
+	storer, err = s.Storer.Module("foo")
```

Comment:
What is the point in calling Module twice?
(explain this in the name of the test)

Actually what is the point of this whole test?